node/res/genesis_info/staging.json

```diff
@@ -31,15 +23,14 @@
     ],
     "council": [
       "5FZP2oqDBBWzaKp8STUQKTvSo2Y1UD68briboWreLiVAxJr1",
-      "5EJLoe5Uaj8U7jTxJwiDCP1kNSnHu4Buw8pdkpm136QRiAEC"
+      "5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY"
     ],
     "technicalCommittee": [
       "5FZP2oqDBBWzaKp8STUQKTvSo2Y1UD68briboWreLiVAxJr1",
-      "5EJLoe5Uaj8U7jTxJwiDCP1kNSnHu4Buw8pdkpm136QRiAEC"
+      "5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY"
     ],
```

Comment:
Why are council and tech committee members changed as well?

Reply:
> Why are council and tech committee members changed as well?

I replaced the second collator with the root account.
I could simply delete it too if required